themes/default/content/docs/guides/scim/okta.md

```diff
@@ -100,6 +100,14 @@ Now that the SCIM connector knows how to connect to Pulumi, you need to adjust t
 5. Click **Save Mappings**.
 6. Click **Apply Updates**.
 
+## Adjusting the Okta to Pulumi Username Mappings
+
+It is important that usernames are not changed after a user is associated with a Pulumi application. To disable Okta from pushing username updates to Pulumi, perform the following steps:
```

Comment:
I suggest we make a slight edit to just say that usernames are immutable to clarify why users should do this:

> Pulumi usernames are immutable and should not be changed after a user is associated with a Pulumi application. To disable Okta from pushing username updates to Pulumi, perform the following steps: